pkg/metadata/labels.go

```diff
@@ -133,14 +133,10 @@ func NewInstanceLabeler(instanceMeta metav1.Object) GenericLabeler {
 
 // NewKroMetaLabeler returns a new labeler that sets the OwnedLabel,
 // KroVersion, and ControllerPodID labels on a resource.
-func NewKroMetaLabeler(
-	kroVersion string,
-	controllerPodID string,
-) GenericLabeler {
+func NewKroMetaLabeler() GenericLabeler {
```

Comment:
What's the experience when running tests locally (in a KIND cluster, for example).

Are the labels set correctly? I looked at the makefile and I _think_ they will be?
What about in envtest?

Reply:
I did some tests locally for this - I like the idea of injecting some in envtest, i'll send another commit add more tests here.